src/pages/Home/Movies/MovieForm/MovieForm.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
+import Row from '../../../../components/Skeleton/Row';
+import Col from '../../../../components/Skeleton/Col';
+import InputField from '../../../../components/forms/InputField';
+import Button from '../../../../components/Button/Button';
+import MultipleSelect from '../../../../components/forms/MultipleSelect/MultipleSelect';
+import MultipleSelectOption from '../../../../components/forms/MultipleSelect/MultipleSelectOption';
+import './MovieForm.css';
+
+class MovieForm extends Component {
+  constructor(props) {
+    super(props);
+    this.defaultFormData = this.props.formData;
+    this.state = {
+      ...this.defaultFormData
```

Comment:
itt nem lehetne simán ...this.props.formData ?
akkor nem lenne szükség kiszedni külön this.defaultFormData propertybe.

Reply:
Válasz írás közben nekem is feltűnt, hogy miért nem kel this.defaultFormData.
Feleslegesen tovább vittem egy ötletet a form visszaállítás miatt...😩
Javítom majd.

Reply:
Fixed in [637ca5c](https://github.com/rkasza/netflix-roulette/commit/637ca5cafaf1f23e533556f49486dea67f497240)